meson_scripts/init.py

```diff
@@ -67,7 +67,7 @@ def init_submodules(
     github_repo_ninja = "https://github.com/ninja-build/ninja"
     sha_version_mpp = "5ff579f43781cae07411e5ab46291c9971536be6"
     github_repo_mpp = "https://github.com/mutationpp/Mutationpp"
-    sha_version_coolprop = "0ce42fcf3bb2c373512bc825a4f0c1973a78f307"
+    sha_version_coolprop = "bafdea1f39ee873a6bb9833e3a21fe41f90b85e8"
```

Comment:
It doesn't matter what the original hash commit is. We just need to make sure SU2 will update CoolProp to the latest version whenever we install and compile SU2.